File: cpp/bc2029.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main()
{
	double v, d, area, al;
	
	while(cin >> v >> d)
	{
		area = pow(d/2, 2) * 3.14;
		al = v  / area;
		
		cout << fixed << setprecision(2) << "ALTURA = " << al << "\nAREA = " << area << endl;
	}
}

File: python/bc2029.py
from math import pow

while True:
    try:
        v = float(input())
        d = float(input())
    except EOFError:
        break
    else:
        area = pow(d/2, 2) * 3.14
        print(f"ALTURA = {v / area:.2f}\nAREA = {area:.2f}")
